### DISCUSSION TITLE
Requesting Update qtile for archlinux

### DISCUSSION THREAD

**Discussion by bkkavin** [qtile/qtile#3018] [General]
Please Update qtile for archlinux

**Comment by elParaguayo** [comment]
We'll release a new version soon (then it's up to the Arch packagers to package this for you), but what's the rush?

You could always use `qtile-git` from the AUR if you want to have the latest.

**Comment by bkkavin** [comment]
appreciate the quick response. Im getting target not found for qtile-git

**Reply by elParaguayo** [reply]
It's in the AUR: https://aur.archlinux.org/packages/qtile-git

It's not in the main Arch repos.

If the AUR file isn't working, let us know.

**Comment by bkkavin** [comment]
python-setuptools 
python-setuptools-scm 
python-bowler .
couldn't find these dependencies in aur.
thank you for the quick response again. 
(I'm new to linux. sry for dumb questions)

**Reply by elParaguayo** [reply]
Those packages are all in the official repos. You can install them with pacman.

**Reply by bkkavin** [reply]
==> ERROR: 'pacman' failed to install missing dependencies.
==> Missing dependencies:
  -> python-setuptools
  -> python-setuptools-scm
  -> python-bowler
==> ERROR: Could not resolve all dependencies.

**Reply by elParaguayo** [reply]
That makes no sense and is getting beyond the support we can provide here. I can't see any reason why pacman would fail to install those packages. You may need to ask on the Arch forum about that.

What happens if you just run `sudo pacman -S python-setuptools python-setuptools-scm python-bowler` before trying to install `qtile-git`?

**Reply by bkkavin** [reply]
I figured. Sorry.  This is the what came up when I went into to directory which had PKGBUILD and ran makepkg -s.
